cdap-distributions/src/packer/cdap-sdk-ubuntu12-with-uri.json

```diff
@@ -126,8 +126,10 @@
       "skip_install": true,
       "json": {
         "cdap": {
-          "version": "3.5.0-SNAPSHOT-1",
+          "comment": "DO NOT PUT SNAPHOT IN THE VERSION BELOW, THIS CONTROLS CDAP COOKBOOK CODE",
+          "version": "3.5.0-1",
```

Comment:
I think the build will fail if it tries to build this VM, since cdap_cookbook with this version doesn't exist.
@wolf31o2 ?